Gopkg.lock

```diff
@@ -178,6 +178,14 @@
   revision = "0ca988a254f991240804bf9821f3450d87ccbb1b"
   version = "v1.3.0"
 
+[[projects]]
+  branch = "master"
+  digest = "1:e2b86e41f3d669fc36b50d31d32d22c8ac656c75aa5ea89717ce7177e134ff2a"
+  name = "github.com/golang/glog"
```

Comment:
I wonder why this dependency was added?

Reply:
it seems to be from pkg, [e2e_flags.go](https://github.com/knative/pkg/commit/d9e3e244a6c044379b71d90b7440daad5b7910f3)